index.html

```diff
@@ -56,7 +60,8 @@ <h3>Haiku</h3>
 
                 <h3>Public Domain Audiobook</h3>
                 <audio controls>
-                    <source src="https://ia801508.us.archive.org/14/items/twelvetypes_2010_librivox/twelvetypes_01_chesterton_128kb.mp3" type="audio/mp3">
+                    <source src="https://ia801508.us.archive.org/14/items/twelvetypes_2010_librivox/twelvetypes_01_chesterton_128kb.mp3" type="audio/mpeg">
+                    <p><a href="https://ia801508.us.archive.org/14/items/twelvetypes_2010_librivox/twelvetypes_01_chesterton_128kb.mp3">Download the audiobook</a></p>

```

Comment:
Careful with MIME types. They're not always the same as the file extension
Fallback link as per lab instructions